network/network_windows.go

```diff
@@ -27,11 +28,10 @@ type route interface{}
 func (nm *networkManager) newNetworkImpl(nwInfo *NetworkInfo, extIf *externalInterface) (*network, error) {
 	// Initialize HNS network.
 	hnsNetwork := &hcsshim.HNSNetwork{
-		Name:               nwInfo.Id,
-		NetworkAdapterName: extIf.Name,
```

Comment:
We should not remove network adapter name. Please separate this change from the time delay you have added below. We can discuss this on a separate PR/Issue.

Reply:
The current usage is still not correct. If this nic name passed is a vnic part of an existing switch, this call is going to fail.

I'll revert this change, but will add another check to skip host vnic.